modules/insticatorBidAdapter.js

```diff
@@ -73,10 +73,26 @@ function buildImpression(bidRequest) {
     insticator: {
       adUnitId: bidRequest.params.adUnitId,
       adUnitName: bidRequest.params.adUnitName,
-      impressionType: bidRequest.params.impressionType
     },
   }
 
+  /**
+   * set impression type using header bidding wrapper's API
+   * this is Insticator header bidding wrapper specific
```

Comment:
I think this logic should be in our ad code. Do we want our prebid bid adapter to have code that depends on our header code implementation?  I think we'd be better off just setting `ext.insticator.impressionType` in our ad code.

Reply:
Chatted with @insticator-biY about this.  As this is really meant to be used as part of our internal instiBid build this seems less of an issue.  Let's go forward with this and see how it all works.